src/java/htsjdk/tribble/Tribble.java

```diff
@@ -51,8 +52,53 @@ public static String indexFile(String filename) {
      * @param file
      * @return
      */
-    public static File indexFile(File file) {
-        return new File(file.getAbsoluteFile() + STANDARD_INDEX_EXTENSION);
+    public static File indexFile(final File file) {
+        return indexFile(file.getAbsoluteFile(), STANDARD_INDEX_EXTENSION);
     }
 
+
+    /**
+     * Return the name of the tabix index file for the provided {@code filename}
+     * Does not actually create an index
+     * @param filename
+     * @return
+     */
+    public static String tabixIndexFile(final String filename) {
+        return indexFile(filename, TabixUtils.STANDARD_INDEX_EXTENSION);
+    }
+
+    /**
+     * Return the File of the tabix index file for the provided {@code file}
+     * Does not actually create an index
+     * @param file
```

Comment:
@nh13 the javadoc is just copied from the existing one above (with the extra word "tabix" included in the description).  I was trying to be consistent.
Happy to fix them both, but I'm not 100% sure what you want them to be.